Sprint-3/reading-list/script.js

```diff
@@ -1,3 +1,28 @@
+function readingList(books) {
+  const readingListContainer = document.getElementById("reading-list");
+
+  books.forEach((book) => {
+    const listItem = document.createElement("li");
```

Comment:
well done this is a really good way of making sure the code is dynamically generating the output regardless of list size